pipeline/terraform/test-environment/README.md

```diff
@@ -0,0 +1,35 @@
+# Terraform
+This directory contains the [Terraform](https://www.terraform.io/) configurations used to deploy instances of the MHS
+application to AWS' ECS service for integration testing as part of the build pipeline.
+
+This configuration will create a new ECS task definition referencing the Docker container image provided as a parameter
+and then create a new ECS service in a pre-existing cluster (again, provided as a parameter) which will run a single
+instance of the container image. 
+
+## Pre-requisites
+In order to function, this configuration expects several components to exist in the AWS account being used: 
+
+- An IAM role for the 'Elastic Container Service Task' trusted entity with the built-in
+  `AmazonECSTaskExecutionRolePolicy`. You can use the `ecsTaskExecutionRole` created for you automatically by AWS when
+  creating a task definition from the console, if available. This ARN of this role is passed as the
+  `task_execution_role` input variable.
+- An IAM role for the EC2 instance/etc. running the build pipeline. This must grant the `ecs:RegisterTaskDefinition`
+  ,`ecs:CreateService`, `iam:PassRole`, `ecs:DescribeTaskDefinition`, `ecs:DeregisterTaskDefinition`, 
+  `ecs:DescribeServices`, `ecs:UpdateService` & `ecs:DeleteService` permissions to the build machine.
```

Comment:
What's the difference between this list and the one in `pipeline/README.md`, both say they're for the build pipeline worker thing.

Reply:
This list I think Garreth added specifically for Terraform to start, the permissions in `pipeline/README.md` also contains the permission for Packer to publish to ECR.